sql/core/src/main/scala/org/apache/spark/sql/parquet/ParquetRelation.scala

```diff
@@ -151,15 +151,20 @@ private[sql] object ParquetRelation {
    *
    * @param pathString The directory the Parquetfile will be stored in.
    * @param attributes The schema of the relation.
+   * @param overwrite  Overwrite the existed file path,
+   *                      If it's false, an exception will raise if the path already existed,
```

Comment:
the alignment is off here. Just add a space after overwrite, and don't do vertical alignment.